src/ontology/mondo-edit.obo

```diff
@@ -134098,21 +134098,20 @@ is_a: MONDO:0003847 {source="MESH:C562461/inferred"} ! hereditary disease
 [Term]
 id: MONDO:0007129
 name: tooth agenesis, selective, 1
-def: "Any tooth agenesis in which the cause of the disease is a mutation in the MSX1 gene." [MONDO:patterns/disease_series_by_gene]
+def: "Any tooth agenesis in which the cause of the disease is a mutation in the MSX1 gene characterized by varying severity of tooth agenesis that may be seen in combination with orofacial clefting in some individuals." [https://clinicalgenome.org/affiliation/40059/, MONDO:patterns/disease_series_by_gene, PMID:27699475, PMID:35877402]
 subset: gard_rare {source="GARD:18244"}
 subset: nord_rare {source="MONDO:NORD"}
 subset: rare
 synonym: "hypodontia/oligodontia 1" RELATED [OMIM:106600]
-synonym: "hypodontia/oligodontia with orofacial cleft" RELATED [OMIM:106600]
 synonym: "MSX1 tooth agenesis" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
+synonym: "MSX1-related tooth agenesis with or without orofacial cleft" EXACT [https://clinicalgenome.org/affiliation/40059/]
```

Comment:
Thanks! I gather that adding it as a synonym is what's needed for "Preferred gene-related syndrome label".

Reply:
yes, they asked for the new term with a new label, but the term already existed, so I added their label as a synonym.